common/kernel/nvme.h

```diff
@@ -811,6 +811,27 @@ typedef enum
     NVM_CMD_DATASET_MGMNT = 0x09
 } nvm_cmd_opcodes_e;
 
+/**
+ * @brief PCIe register Opcodes
+ */
+typedef enum
+{
+    /**
+     * @brief register read.
+     */
+    PCI_REG_READ = 0x00,
+    /**
+     * @brief register write.
+     */
+    PCI_REG_WRITE = 0x01,
+} ioctl_regop_e;
+
+struct pcie_regs_rw {
+    vmk_uint32 offset;
+    ioctl_regop_e op;
+    vmk_uint16 size;
```

Comment:
What's the purpose for the field "size"? Seems it is only used in the assertion in Line: 480  of file common/kernel/nvme_ctrlr_mgmt